tests/kafkatest/sanity_checks/test_console_consumer.py

```diff
@@ -28,18 +30,23 @@ class ConsoleConsumerTest(Test):
     def __init__(self, test_context):
         super(ConsoleConsumerTest, self).__init__(test_context)
 
+        self.test_context = test_context
```

Comment:
`self.test_context = test_context` is not necessary

every subclass of `Test` has this assigned as long as you make the proper call to `super(...).__init__(...)` (which you did)